code/modules/medical_machinery/machines/advanced_medical_machinery.dm

```diff
@@ -59,23 +60,24 @@
 	name = "Thermal Stabilizer"
 	desc = "Stabilizes free plasma particles in inorganic bodies, causing them to not burn. Uses massive amounts of electricity.This model seems to be very old."
 	icon_state = "thermal_stabilizer"
-	var/stabilization_rate = 15
+	var/stabilization_rate = 10
 
 /obj/machinery/medical/thermal/RefreshParts()
 	for(var/obj/item/stock_parts/micro_laser/ML in component_parts)
-		stabilization_rate *= ML.rating
+		stabilization_rate = inital(stabilization_rate) * ML.rating
```

Comment:
This does not support multiple of the same part. You should follow what other machines do.